node-api/handlers/beacon/genesis_test.go

```diff
@@ -134,8 +134,7 @@ func TestGetGenesis(t *testing.T) {
 
 			// setup test
 			backend := mocks.NewBackend(t)
-			h := beacon.NewHandler(backend)
-			h.SetLogger(noop.NewLogger[log.Logger]())
```

Comment:
This is one of the points of this PR. Just pass the logger immediately in the handler constructor instead of passing the loggers right after